label-application-rule-config-service-impl/src/test/java/org/hypertrace/label/application/rule/config/service/LabelApplicationRuleConfigServiceImplTest.java

```diff
@@ -261,9 +261,11 @@ private LabelApplicationRule createCompositeRule() {
 
   private LabelApplicationRuleData.Action buildAction() {
     return LabelApplicationRuleData.Action.newBuilder()
-        .setEntityType("API")
+        .addAllEntityTypes(List.of("API"))
```

Comment:
nit: addEntityType (repeated types take a single or collection, so no need to wrap to match the collection api)